board/rockchip/evb_rk3588/evb_rk3588.c

```diff
@@ -100,33 +103,45 @@ int board_usb_cleanup(int index, enum usb_init_type init)
 
 #ifdef CONFIG_ID_EEPROM
 static struct variant_def variants[] = {
-	{"rockchip,rk3588", 300, 380, "rockchip/rk3588s-radxa-e54c.dtb"},
-	{"rockchip,rk3588", 980, 1060, "rockchip/rk3588-rock-5t.dtb"},
-	{"rockchip,rk3588", 1650, 1730, "rockchip/rk3588s-radxa-e52c.dtb"},
-	{"rockchip,rk3588", 2360, 2440, "rockchip/rk3588s-rock-5c.dtb"},
-	{"rockchip,rk3588", 3370, 3450, "rockchip/rk3588s-rock-5d.dtb"},
-	{"rockchip,rk3588", 4050, 4130, "rockchip/rk3588-rock-5b-plus.dtb"},
+	{"rockchip,rk3588", 300, 380, 0, -1, "rockchip/rk3588s-radxa-e54c.dtb"},
+	{"rockchip,rk3588", 1650, 1730, 0, -1, "rockchip/rk3588s-radxa-e52c.dtb"},
+	{"rockchip,rk3588", 2360, 2440, 0, -1, "rockchip/rk3588s-rock-5c.dtb"},
+	{"rockchip,rk3588", 3370, 3450, 0, -1, "rockchip/rk3588s-rock-5d.dtb"},
+	{"rockchip,rk3588", 4050, 4130, 0, -1, "rockchip/rk3588-rock-5b-plus.dtb"},
+	{"rockchip,rk3588", 980, 1060, 0, 40, "rockchip/rk3588-rock-5t.dtb"},
+	{"rockchip,rk3588", 3040, 3120, 640, 720, "rockchip/rk3588-rock-5t-industrial.dtb"},
```

Comment:
```suggestion
	{"rockchip,rk3588",  300,  380,    0,   -1, "rockchip/rk3588s-radxa-e54c.dtb"},
	{"rockchip,rk3588",  980, 1060,    0,   40, "rockchip/rk3588-rock-5t.dtb"},
	{"rockchip,rk3588", 1650, 1730,    0,   -1, "rockchip/rk3588s-radxa-e52c.dtb"},
	{"rockchip,rk3588", 2360, 2440,    0,   -1, "rockchip/rk3588s-rock-5c.dtb"},
	{"rockchip,rk3588", 3040, 3120,  640,  720, "rockchip/rk3588-rock-5t-industrial.dtb"},
	{"rockchip,rk3588", 3370, 3450,    0,   -1, "rockchip/rk3588s-rock-5d.dtb"},
	{"rockchip,rk3588", 4050, 4130,    0,   -1, "rockchip/rk3588-rock-5b-plus.dtb"},
```
从小到大按顺序排列吧，另外现在5t和5t industrial都没有共享hw id啊，那也可以不加bom id检测吧？

Reply:
修改完毕，hwid是一样的，已改正